src/Command/BaseCommand.php

```diff
@@ -91,6 +92,9 @@ public function getErrorListener()
      * @param string $mode
      *
      * @return BackendInterface
+     *                          NEXT_MAJOR: Remove this method
```

Comment:
```suggestion
     * NEXT_MAJOR: Remove this method
```

Reply:
@VincentLanglet when I remove this space from begginner I get the lint broken :/ 

https://github.com/sonata-project/SonataPageBundle/runs/7130759831?check_suite_focus=true#step:5:27

Reply:
I think it's because you need a blank line around the @return phpdoc.
The linter is thinking you're adding a multi-line comment here.

Reply:
yes it's the problem, thanks :)